{{cookiecutter.project_slug}}/.github/workflows/expo-pr.yml

```diff
@@ -0,0 +1,51 @@
+name: Expo Publish PR
+on: [pull_request]
+jobs:
+  Setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+      - name: Set URL for PR
+        if: success() && steps.findPr.outputs.number
+        run: echo "BASE_URL=https://example-pr-${PR}.herokuapp.com/" >> $GITHUB_ENV
```

Comment:
check out the latest update to the Cypress Github Action
https://github.com/thinknimble/tn-spa-bootstrapper/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/cypress.yml#L19

Heroku will create a deployment event, which you can use the trigger a workflow like this one. That also means it provides the URL needed to call it, so you no longer need to use `jwalton/gh-find-current-pr@v1`

I imagine right now you are having problems where this action fails unless/until a deployment has succeeded and is thus a bit unstable

Reply:
ughh this failed for me on Worthy need to try it again and will update!

Reply:
The `findPr` action often fails the first time a PR is opened. So you either have to push another commit or manually re-trigger it.
Another reason to move away from it and just trigger on a deploy event.